### PR TITLE
Helm chart ships the pod security policy required to run LH in CIS hardened clusters

### PR DESCRIPTION
**Pull request by janeczku**
This PR allows Longhorn to successfully run in a CIS hardened Cluster using the Helm chart install. See https://github.com/longhorn/longhorn/issues/608

To test:
1. Create a cluster hardened according to CIS Kubernetes benchmark
2. Deploy with current release Helm chart -> Longhorn pods fail to start / crashloop
3. Deploy with Helm chart per this PR -> Longhorn successfully installed

### CODE REVIEW

**Comment by kursattokpinar** [comment]
Is PSP used here designed with 'least privileges'?

**Comment by janeczku** [comment]
@kursattokpinar PSP here uses recommended "restricted" CIS Kubernetes benchmark policy as starting point,  extended only with the least privileges required to run Longhorn pods.

**Comment by yasker** [comment]
@janeczku In fact, Kubernetes community is planning to depreciate the PSP soon, see https://github.com/kubernetes/enhancements/issues/5#issuecomment-637066475

File: chart/values.yaml [hunk 1]
```diff
@@ -114,3 +114,7 @@ ingress:
   # - name: longhorn.local-tls
   #   key:
   #   certificate:
+
+# Setup pod security policy in the Longhorn namespace to allow privileged pods
+podSecurityPolicy:
+  enabled: true
```
Comment (ttpcodes) [inline chart/values.yaml:120]: Are there any other fields that we'd put under the `podSecurityPolicy` value? If not, I think it makes sense to just condense this key into something like `enablePSP` for the user.

Comment (janeczku) [reply chart/values.yaml]: Modified per your suggestion.

**Pull request merged**